### DISCUSSION TITLE
Xxl_job_log table Slow query[Bug]

### DISCUSSION THREAD

**Discussion by BigbigY** [apache/skywalking#9167] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

OAP server (apache/skywalking)

### What happened

DBA student feedback xxl_job_log a large number of slow query, log table has millions of data

### What you expected to happen

Xxl_job Uses the latest version

### How to reproduce

Check the Xxl_job_log

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
Why is this a concern of the SkyWalking community?
